packages/amplication-server/src/core/prismaSchemaParser/prismaSchemaParser.service.ts

```diff
@@ -811,6 +825,25 @@ export class PrismaSchemaParserService {
             .field(field.name)
             .removeAttribute(DEFAULT_ATTRIBUTE_NAME);
       });
+
+      const hasIdField = modelFields.some(
+        (field) =>
+          field.attributes?.some((attr) => attr.name === ID_ATTRIBUTE_NAME) ??
+          false
+      );
+
+      // add the id field if it doesn't exist. The type is the default type for id field in Amplication - String
+      if (!hasIdField) {
+        builder
+          .model(model.name)
+          .field(ID_FIELD_NAME, "String")
+          .attribute(ID_ATTRIBUTE_NAME);
```

Comment:
Can we add a test to cover this scenario?
I'm not sure if this logic would work in case of multiple unique attributes
```ts
datasource db {
            provider = "postgresql"
            url      = env("DB_URL")
          }
          
          generator client {
            provider = "prisma-client-js"
          }
          
          model Example {
            id    Int @id @default(0)
            value Int    @default(123)    
          }
  
          model Test {
            test_id    String @unique @default("mock_id")
            value2     Int  @unique
          }
```

Reply:
Just saying, this is not a new logic, I just moved it

Reply:
[008158c](https://github.com/amplication/amplication/pull/6677/commits/008158ce97605f72b8bfb295be30232083078d19)